k8s/upgrades/1.0.0-1.1.0/jiva/example-jiva-upgrade-job.yaml.tpl

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: jvupg-100111-<@pv-name>
+spec:
+  backoffLimit: 0
+  template:
+    spec:
+      serviceAccountName: <@openebs-service-account>
+      containers:
+      - name:  upgrade
+        image: openebs/m-upgrade:ci-1.1.x-072501
+        args: 
+        - "<@from-version>"
+        - "<@to-version>"
+        - "jivaVolume"
+        - "<@pv-name>"
+        - "<@openebs-namespace>"
+        tty: true 
+      restartPolicy: Never
```

Comment:
Should we have some reconciliation here so that if job fails due to some reason like network issue or something, it retries.

Reply:
good point. changed it.